webapp/templates/rewriterule-create.xhtml

```diff
@@ -0,0 +1,176 @@
+            <input type="hidden" property="calli:missing" content="?missing" value="{?missing}"/>
+            <input type="hidden" property="calli:gone" content="?gone" value="{?gone}"/>
+            <input type="hidden" property="calli:illegal" content="?illegal" value="{?illegal}"/>
+            <input type="hidden" property="calli:post" content="?post" value="{?post}"/>
+            <input type="hidden" property="calli:put" content="?put" value="{?put}"/>
+            <input type="hidden" property="calli:patch" content="?patch" value="{?patch}"/>
+            <input type="hidden" property="calli:delete" content="?delete" value="{?delete}"/>            
+        </div>
+                
+        <div class="row">
+            <div class="col-sm-12">
+                <label>Rules
+                    <a href="javascript:void(0)" title="More" onclick="addRule('#rules')" class="glyphicon glyphicon-plus" />
+                </label>
+                <table id="ruleTable" class="table-condensed col-sm-12">
+                    <thead>
+                        <tr>
+                            <th class="col-sm-5" colspan="2">Request</th>
+                            <th class="col-sm-6" colspan="2">Response</th>
+                            <th class="col-sm-1"></th>
+                        </tr>
+                        <tr>
+                            <th class="col-sm-2">Methods</th>
+                            <th class="col-sm-3">Pattern</th>
+                            <th class="col-sm-2">Method/Status</th>
+                            <th class="col-sm-4">Template</th>
+                            <th class="col-sm-1"></th>
+                        </tr>
+                    </thead>
+                    <tbody id="rules">
+                    </tbody>
+                    <tfoot style="display:none">
+                        <tr class="rule">
+                            <td>
+                                <select multiple="multiple" class="form-control requestMethod">
+                                    <option value="GET" selected="selected">GET</option>
+                                    <option value="POST">POST</option>
+                                    <option value="PUT">PUT</option>
+                                    <option value="PATCH">PATCH</option>
+                                    <option value="DELETE">DELETE</option>
+                                </select>
+                            </td>
+                            <td>
+                                <input type="text" class="form-control requestPattern" value=".*" onchange="" placeholder=".*"/>
+                            </td>
+                            <td>
+                                <select class="form-control responseType">
+                                    <!-- GET responses -->
+                                    <optgroup label="GET responses">
```

Comment:
Copy (200), Missing (404), Gone (410), and Illegal (451) are system requests that will retrieve the target via HTTP GET and return the result (other than Copy, chang the response status). Perhaps the label "Safe requests" would be better here?